frontend/packages/dev-console/src/components/topology/components/nodes/MonitoringAlertsDecorator.tsx

```diff
@@ -46,16 +46,21 @@ const MonitoringAlertsDecorator: React.FC<MonitoringAlertsDecoratorType> = ({
       .fireEvent(SELECTION_EVENT, [workload.getId()]);
   };
 
+  if (
+    firingAlerts.length === 0 ||
+    severityAlertType === AlertSeverity.None ||
+    severityAlertType === AlertSeverity.Info
```

Comment:
As this is repeating at multiple places. Can we create util for this? So, it will easy to maintain also.